common/namespace/cache.go

```diff
@@ -510,31 +528,6 @@ UpdateLoop:
 	return nil
 }
 
-func (c *namespaceCache) checkAndContinue(
```

Comment:
Was this just unused?

Reply:
Yup. I didn't dig through history so I don't know what it was for and when it stopped being used...

Oh, I see now, in #1016: there was previously a different force-refresh mechanism that worked synchronously in `getNamespace`, that was removed. I don't think this mechanism has the same problem that one did, and anyway is only used from tests.

Reply:
Good argument for activating some linters as part of the build